.gitignore

```diff
@@ -0,0 +1,2 @@
+.DS_Store
+*/.DS_Store
```

Comment:
These new lines warnings always get me. Sometimes I wish gh wouldnt' highlight them.

I usually keep DS_Store in my `~/.gitignore_global` ([configured like so](https://github.com/matschaffer/dotfiles/blob/master/gitconfig#L42-L43)) but nbd l if you prefer to have them in repo.